WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -163,11 +163,15 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
     }
 
     func configureNavigationButtons() {
+        addButton.accessibilityLabel = "Add new post"
+        postsViewButtonItem.accessibilityLabel = isCompact ? "Switch to expanded list view" : "Switch to compact list view"
```

Comment:
It looks like you only need to do this once in `showCompactOrDefault()` which is called on `viewDidLoad` and whenever `isCompact` value is changed. The less code, the better. What do you think? 🙂 

As for the `accessibilityLabel`, this is a bit tricky because it's a toggle button. Changing the label can be confusing though. The user might think that it's a different button. My suggestion would be to have a static `accessibilityLabel`, `“List style”`, so the user will be able to identify that it's the same button. And then change the `accessibilityValue`. 

Layout Style | `accessibilityLabel` | `accessibilityValue`
--------|-------|---
Expanded        |       List style | expanded 
Compact | List style | compact

The label and the values will still be spoken by VoiceOver.

- [ ] Move to `showCompactOrDefault()`
- [ ] Use `accessibilityValue`

Reply:
I did write that line twice, because I thought it was possible that the user might have left the app on a certain `compact` or `expanded` view, and they might want to return to their choice of view. So it calculates once on `viewDidLoad` and when the toggle is hit. Is this not necessary? 

I'll work on the other changes as well!